### PR TITLE
94 local highscore

### LINKED ISSUE #94: Personal highscore

### PR DESCRIPTION
**Pull request by mpsijm**
Will close #94

This feature is about implementing Local highscore.
State has a public method `checkHighscore()` which changes the highscore when the current score is higher than the highscore.
This method is called when the Player dies (this is done in the InternalTicker).
The highscore is displayed in the Game GUI (ExternalTicker) and Death Popup.

Please review!

<feature,user-interface,interface

### CODE REVIEW

File: src/test/java/nl/tudelft/ti2206/group9/level/StateTest.java [hunk 1]
```diff
@@ -79,5 +79,18 @@ public void testSetCoins() {
 		State.setCoins(1);
 		assertEquals(1, State.getCoins());
 	}
+	
+	@Test
+	public void testGetHighScore() {
```
Comment (thervh70) [inline src/test/java/nl/tudelft/ti2206/group9/level/StateTest.java]: This method does not only test the functionality of getHighscore(). Naming should be changed.

< naming, review, testing

**Comment by thervh70** [comment]
Now it looks fine to me. Please merge!

< review

**Comment by mdingjan** [comment]
Looks nice! :) Please merge.

<review

**Comment by MathiasMeuleman** [comment]
Looks oke! Works nice :)

Will merge

<review

**Pull request merged**